sample-apps/delivery-customizations/.gitignore

```diff
@@ -26,3 +26,7 @@ extensions/*/build
 
 # Node library SQLite database
 web/database.sqlite
+
+# Liquid-generated files
+web/metafields.js
+extensions/delivery-customization/input.graphql
```

Comment:
Looks like this shouldn't be checked in because it also excludes it for partners when they clone it with the CLI 😬